src/CatalogService/src/CatalogService.Api/appsettings.json

```diff
@@ -7,7 +7,10 @@
     },
     "AllowedHosts": "*",
     "UseInMemoryDatabase": false,
-    "ConnectionStrings": {
-        "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=CatalogService;Trusted_Connection=True;MultipleActiveResultSets=true;"
-    }
+  "ConnectionStrings": {
```

Comment:
The connection strings should not be added to the source control

Reply:
Fixed, moved to secrets.